firmament/models.py

```diff
@@ -58,14 +58,13 @@ class Proof(models.Model):
     class Meta:
         ordering = ('-timestamp',)
 
-    def save(self, generate_pdf=False, *args, **kwargs):
+    def save(self, *args, **kwargs):
         super(Proof, self).save(*args, **kwargs)
-        if generate_pdf:
-            from .tasks import generate_proof_pdf
-            print("CALLING TASK")
-            generate_proof_pdf.apply_async([self.pk])
-            if settings.CELERY_ALWAYS_EAGER:
-                self.refresh_from_db()
+        from .tasks import generate_proof_pdf
```

Comment:
previously conditional was checking if `self` had `pk` assigned, and therefore only calling generate_proof_pdf if the proof instance was new and not yet saved once. Since we have to regenerate the pdf proof using cloudconvert on every reupload, removing any such conditional seems to work just fine. Unless I'm missing something big here?